test/interval_tests/construction.jl

```diff
@@ -43,11 +43,11 @@ using Base.Test
 
     # Disallowed conversions with a > b
 
-    @test_throws ArgumentError Interval(2, 1)
-    @test_throws ArgumentError Interval(big(2), big(1))
-    @test_throws ArgumentError Interval(BigInt(1), 1//10)
-    @test_throws ArgumentError Interval(1, 0.1)
-    @test_throws ArgumentError Interval(big(1), big(0.1))
+    @test_throws ArgumentError interval(2, 1)
+    @test_throws ArgumentError interval(big(2), big(1))
+    @test_throws ArgumentError interval(BigInt(1), 1//10)
+    @test_throws ArgumentError interval(1, 0.1)
+    @test_throws ArgumentError interval(big(1), big(0.1))
 
```

Comment:
If my comment about `interval` is correct (`interval` checks the validity and `Interval may not), I think it is a good idea to test the differences among `Interval` and `interval` when their behavior differs.